### DISCUSSION TITLE
How to add own functions for matchers?

### DISCUSSION THREAD

**Discussion by call-me-matt** [casbin/casbin-mesh#71] [Q&A]
Hello,

I would like to add my own mqttMatcher. How can I add my [own function](https://casbin.org/docs/function/#how-to-add-a-customized-function) to the enforcer with casbin-mesh?

Thanks

**Comment by hsluoyz** [comment]
@call-me-matt send a github issue instead

**Reply by call-me-matt** [reply]
OK, [done](https://github.com/casbin/casbin-mesh/issues/72#issue-2187897048).
I was not sure if it is an issue or just missing documentation.
